### PR TITLE
Support paiTrainingService on windows

### PR DESCRIPTION
**Pull request by SparkSnail**

### CODE REVIEW

**Comment by scarlett2018** [comment]
For #1075

File: test/generate_ts_config.py [hunk 1]
```diff
@@ -73,7 +73,7 @@ def update_training_service_config(args):
 
 def convert_command():
     '''convert command by platform'''
-    if sys.platform != 'win32':
+    if sys.platform != 'win32' or args.ts:
```
Comment (chicm-ms) [inline test/generate_ts_config.py]: why add this?

File: src/nni_manager/main.ts [hunk 2]
```diff
@@ -169,4 +169,12 @@ process.on('SIGTERM', async () => {
...
         process.exit(hasError ? 1 : 0);
     }
+}
+
+process.on('SIGBREAK', async () => {
+    await cleanUp();
+})
+
+process.on('SIGTERM', async () => {
+    await cleanUp();
```
Comment (QuanluZhang) [inline src/nni_manager/main.ts]: this has different logic from pr 1073

File: test/generate_ts_config.py (hunk 1 above)
Comment (SparkSnail) [reply test/generate_ts_config.py]: this function is executed only if the platform if local, refactor logic,

File: src/nni_manager/main.ts (hunk 2 above)
Comment (SparkSnail) [reply src/nni_manager/main.ts]: fixed, changed to use logic in pr 1073

**Pull request merged**